frontend/src/metabase-lib/temporal_bucket.ts

```diff
@@ -55,11 +55,16 @@ export function defaultTemporalBucket(
 
 type IntervalAmount = number | "current" | "next" | "last";
 
+type TemporalIntervalOptions = {
+  "include-current"?: boolean;
+};
+
 export function describeTemporalInterval(
   n: IntervalAmount,
   unit?: TemporalUnit,
+  opts?: TemporalIntervalOptions,
```

Comment:
nit: if include-current is the only option we can pass it right away without wrapping in `options` object